app/containers/App/actions.js

```diff
@@ -1,11 +1,33 @@
+import ls from 'local-storage';
+
+import parseJwt from 'services/parseJwt';
+import { storeUserId } from 'containers/SignInForm/actions';
 import {
+  STORE_TOKEN,
   SHOW_LOADER,
   HIDE_LOADER,
   GET_USER_SUCCESS,
   GET_USER_FAILURE,
   SIGN_OUT,
 } from './constants';
 
+export const getToken = () => {
+  const lsToken = ls.get('token');
+
+  return dispatch => {
+    dispatch(storeToken(lsToken));
+    if (lsToken !== null) {
+      const { id } = parseJwt(lsToken);
+      dispatch(storeUserId(id));
```

Comment:
Here I think you should also validate the token by making a request to the api to fetch the logged user, then you can store the whole user object in state.